src/pages/billing/Project.js

```diff
@@ -27,9 +51,51 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
   const [billingAccountName, setBillingAccountName] = useState(null)
   const [showBillingModal, setShowBillingModal] = useState(false)
   const [selectedBilling, setSelectedBilling] = useState()
+  const [tab, setTab] = useState(query.tab || 'workspaces')
 
   const signal = Utils.useCancellation()
 
+  const tabToTable = {
+    workspaces: div({ style: { flexGrow: 1, width: '100%' } }, [
+      _.map(([i, workspace]) => h(WorkspaceCard, { workspace, key: i }), Utils.toIndexPairs(_.map(response => response.workspace, _.filter(response => response.workspace.namespace === projectName, workspaces))))
```

Comment:
Two things, for the future:
1. We shouldn't use index as the key, it's unsafe because if something is inserted in the middle of a list, the index and the item won't be matched anymore.
2. When mapping to just get a property, there's a shorthand: `_.map('workspace', workspaces)`